mm/mempool/mempool.c

```diff
@@ -437,7 +433,7 @@ mempool_info_task(FAR struct mempool_s *pool,
       list_for_every_entry(&pool->alist, buf, struct mempool_backtrace_s,
                            node)
         {
-          if (buf->pid == dump->pid ||
+          if (buf->pid == dump->pid || dump->pid == MM_BACKTRACE_ALLOC_PID ||
```

Comment:
```suggestion
          if (dump->pid == buf->pid || dump->pid == MM_BACKTRACE_ALLOC_PID ||
```
optional: Just from a style perspective keeping `dump` as a left value.